From 6425f6499cc0ec12a6af9bb6f2ecf21cfcaebf92 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Tue, 1 Nov 2022 16:06:20 -0400
Subject: [PATCH] util: Fix `-fpermissive` warning

This seems to only be happening with newer gcc (or something
in the glib2 headers changed?).
---
 src/libpriv/rpmostree-util.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libpriv/rpmostree-util.cxx b/src/libpriv/rpmostree-util.cxx
index 623475fe55..8ee6943d7c 100644
--- a/src/libpriv/rpmostree-util.cxx
+++ b/src/libpriv/rpmostree-util.cxx
@@ -436,7 +436,7 @@ rpmostree_str_ptrarray_contains (GPtrArray *strs, const char *str)
   guint n = strs->len;
   for (guint i = 0; i < n; i++)
     {
-      if (g_str_equal (str, strs->pdata[i]))
+      if (g_str_equal (str, (const char *)strs->pdata[i]))
         return TRUE;
     }
   return FALSE;
